services/motion/builtin/builtin.go

```diff
@@ -145,6 +163,12 @@ func (ms *builtIn) Move(
 	constraints *servicepb.Constraints,
 	extra map[string]interface{},
 ) (bool, error) {
+	ms.mu.RLock()
```

Comment:
All these methods need to take the read lock on the mutex, otherwise, they are not thread-safe during reconfiguration.